PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -1920,6 +1920,7 @@ public void RemoveAllNotesClick(Office.IRibbonControl control)
         public void CustomShapeButtonClick(Office.IRibbonControl control)
         {
             InitCustomShapePane();
+            TogglePaneVisibility();

```

Comment:
Okay, changed ShapesLab and AddCustomShape to action framework.
Tests passing on 2013